kafka-connect-redis/src/test/scala/com/datamountaineer/streamreactor/connect/redis/sink/writer/RedisSslTest.scala

```diff
@@ -59,36 +59,69 @@ class RedisSslTest extends WordSpec with Matchers with BeforeAndAfterAll with Mo
     SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG -> truststoreFilePath
   )
 
-  def setupTrustStore(): Unit = {
-    setJvmTrustStore(truststoreFilePath, "jceks")
-  }
-
-  private def setJvmTrustStore(trustStoreFilePath: String, trustStoreType: String): Unit = {
-    new File(trustStoreFilePath).exists shouldBe true
-    System.setProperty("javax.net.ssl.trustStore", trustStoreFilePath)
-    System.setProperty("javax.net.ssl.trustStoreType", trustStoreType)
-  }
-
-  override def beforeAll() = {
-    if (runTests) {
-      setupTrustStore()
-    }
-  }
-
-  override def afterAll() = {
-  }
+//  def setupTrustStore(): Unit = {
```

Comment:
do we need these? should we remove them?

Reply:
its fine